core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,599 @@
+
+---
+
+## Installation
 
 ```bash
-yarn add @dotcms/angular
+npm install @dotcms/angular@next
 ```
 
-## Configuration
-### Provider Setup
-We need to provide the information of our dotCMS instance
+### Peer Dependencies
 
-```javascript
+Make sure to also install:
 
-import { ClientConfig } from '@dotcms/client';
+```bash
+npm install @dotcms/uve@next @dotcms/client@next @dotcms/types@next
+```
 
-const DOTCMS_CLIENT_CONFIG: ClientConfig = {
-    dotcmsUrl: environment.dotcmsUrl,
-    authToken: environment.authToken,
-    siteId: environment.siteId
-};
+---
+
+## Key Concepts
+
+| Term                  | Description                                                          |
+|----------------------|----------------------------------------------------------------------|
+| `pageAsset`          | The actual page layout data used by the renderer                     |
+| `DotCMSPageComponent`| Type for mapping content types to Angular components                 |
+| UVE                  | Universal Visual Editor, dotCMS's editing interface                  |
+| Editable Page        | A page enhanced via `DotCMSEditablePageService` for real-time updates|
+
+---
+
+## API Reference
+
+### Components
+
+#### DotCMSLayoutBody
+
+**Overview**: Component used to render the layout for a DotCMS page, supporting both production and development modes.
+
+**Inputs**:
+
+| Input        | Type                    | Required | Default        | Description                                     |
+|--------------|-------------------------|----------| ---------------|------------------------------------------------|
+| `page`       | `DotCMSPageAsset`      | ✅        | -              | The page asset containing the layout to render  |
+| `components` | `DotCMSPageComponent`   | ✅        | `{}`           | Map of content type → Angular component         |
+| `mode`       | `DotCMSPageRendererMode`| ❌        | `'production'` | Rendering mode ('production' or 'development')  |
+
+**Usage**:
+
+```typescript
+import { DotCMSPageAsset } from '@dotcms/types';
+import { DotCMSLayoutBody } from '@dotcms/angular/next';
+
+@Component({
+    template: `
+        <dotcms-layout-body
+            [page]="pageAsset()"
+            [components]="components()"
+            mode="development"
+        />
+    `
+})
+export class PageComponent {
+    components = signal<DotCMSPageComponent>({
+        Blog: import('./blog.component').then(c => c.BlogComponent)
+    });
+    pageAsset = signal<DotCMSPageAsset | null>(null);
+
+    constructor(private readonly client: DotCmsClient) {}
+
+    ngOnInit() {
+        this.client.page
+            .get({ url: '/my-page' })
+            .then(({ pageAsset }) => {
+                this.pageAsset.set(pageAsset);
+            });
+    }
+}
 ```
 
-Add this configuration to `ApplicationConfig` in your Angular app.
+**Component Mapping**:
+
+* The key `Blog` is the content type variable in dotCMS
```

Comment:
Can we link to dotcms content types maybe there is a variable section.

Reply:
Done! I linked to: https://dev.dotcms.com/docs/content-types#VariableNames